### PR TITLE
adds headers to index pages etc, only if they exist.

### PR DESCRIPTION
**Pull request by sephirothkod**
ref #20 and #21

### CODE REVIEW

**Review by bkiahstroud — COMMENTED** [review]
The referenced tickets both have an Acceptance Criteria re: the "New" button: 

> The importers and exporters index pages will have a header that includes the "New" button that takes the user to a new form 

Are those buttons rendering now with this change?

**Comment by labradford** [comment]
> The referenced tickets both have an Acceptance Criteria re: the "New" button:
> 
> > The importers and exporters index pages will have a header that includes the "New" button that takes the user to a new form
> 
> Are those buttons rendering now with this change?

The new button is inside the page header

> The referenced tickets both have an Acceptance Criteria re: the "New" button:
> 
> > The importers and exporters index pages will have a header that includes the "New" button that takes the user to a new form
> 
> Are those buttons rendering now with this change?

The buttons are inside the page header content

**Comment by bkiahstroud** [comment]
> The buttons are inside the page header content

I see... so that conditional is truthy on Bulkrax views, rendering the buttons, and falsy everywhere else. Am I getting that correct?

File: hydra/app/views/layouts/collection.html.erb [hunk 1]
```diff
@@ -56,6 +56,13 @@
             <div class="alert alert-<%= key %>"><%= value %></div>
         <% end %>
         <div class="row">
+          <% if content_for?(:page_header) %>
```
Comment (labradford) [inline hydra/app/views/layouts/collection.html.erb:59]: Would it make more sense to add this as a conditional in Bulkrax rather than here?

**Pull request merged**